CONTRIBUTING

```diff
@@ -0,0 +1,102 @@
+# How to contribute
+
+Welcome to `digits` library for GoLang.
+
+###### TODO <@joshuanario, insert blurb here about ECKSBEE and the library.>
+###### TODO <Contact Information / Slack Invite>
+
+### Resources
+
+| Resource | Description |
+| --- | --- |
+| [XBRL Standards](https://www.xbrl.org/) | Business Reporting Standard Community. Details XBRL and how it is used, and what to expect with this library.       |
+| [EDGAR Browser](https://edgarbrowser.com) | Open Source EDGAR Browser, leveraging Telefacts. |
+| [Telefacts](https://github.com/ecksbee/telefacts) | XBRL file storage, data types, and renderer. |
+| [ECKSBEE](https://ecksbee.com/) | Our |
+
+###### TODO <consider adding other resources, like were bugs should be reported to, etc.>
+
+## Coding conventions
+
+Start reading our code and you'll get the hang of it. This library is optimized for size and efficiency:
+
+1. `digits.go` contains the only public facing functionality.

```

Comment:
`digits.go` contains the only public facing functionality.

Please remove this.  I don't think it's necessary to formalize this in a CONTRIBUTING file.  Let code reviewers (like myself) call this out or make flexible decisions about this.